recipe/meta.yaml

```diff
@@ -70,3 +96,4 @@ extra:
     - dcherian
     - keewis
     - TomNicholas
+    - kmuehlbauer
```

Comment:
if you split this into a separate PR, we can merge that before merging this PR?

Reply:
Sure!

Reply:
https://github.com/conda-forge/xarray-feedstock/pull/114